### ISSUE TITLE
Would you be interested in a PR for this theme?

### ISSUE THREAD

**Issue by Cldfire** [azerupi/mdBook#223]
Hello,

I recently put together a theme for [various Rustlang-related websites](https://github.com/Cldfire/ayu-rs) based on [this ST3 theme](https://github.com/dempfi/ayu). I was looking into porting it to the upcoming Rust book as well, which is, as you are probably aware, built with this program. I thought it would also be worth asking if you would be interested in a PR to have this theme added to mdBook's out-of-the-box themes?

If not, that's perfectly fine (I understand you already have a blueish dark theme).

**Comment by azerupi** [comment]
Sure! :)
I am totally fine with that, however in the future we should definitely find a better way to support themes. Hardcoding them is not very scalable.. :/

**Comment by Cldfire** [comment]
Cool :) no promises as to when I'll be able to do it, but you'll see something from me in the future.

**Comment by Cldfire** [comment]
Status update: I have started to work on this and should have something to PR shortly (I didn't realize how little work this would be, haha. Spent most of my time so far digging around to get another theme in at all, I see what you mean by hardcoded).

**Comment by Cldfire** [comment]
Another status update! I deeply apologize for the large communication void on this; I came down with something just after I posted the last update, wasn't feeling good, and got bogged down with school / other projects after I recovered.

I just finished implementing a draft (if you will) of what this will look like as part of my Stylish theme:

![doc rust-lang org-stable-book-match html](https://cloud.githubusercontent.com/assets/13814214/26812416/94de8700-4a45-11e7-8978-256c60a534a9.png)

I think it looks pretty good, so I'm going to go ahead and _crosses fingers_ hopefully get it converted over to work as a built-in theme in the next few days or so.

Thanks for being patient with me, and I again apologize for leaving this sitting here for so long :)
